Twig/Extension/FormatterMediaExtension.php

```diff
@@ -76,14 +76,6 @@ public function getTwigExtension()
     }
 
     /**
-     * {@inheritdoc}
-     */
-    public function getName()
```

Comment:
It should work on Twig >= 1.26 as it was marked as deprecated in that version. This PR has added Twig in `composer.json` like this  "twig/twig": "^1.28 || ^2.0" so there should be no problem IMO.